src/components/button/button.api.json

```diff
@@ -217,15 +222,6 @@
                   }
                 ]
               },
-              {
```

Comment:
You kept the `Full-Width Vs. Floating Button` section in the `UX Guidelines`

Reply:
I think we can keep it, FloatingButton is available in public